### PR TITLE
Fix: article attribution, typos, article image replacement

### PR DESCRIPTION
**Pull request by cderr**
- added Discord link to community support initiative
- fixed typo in Wyatt's bio
- added correct image to Teaching Technology Lessons Learned article
- changed image for Relinquishing Our Humanity article
- corrected attribution in Geeks Bearing Grifts
- fixed typo on About page

### CODE REVIEW

File: docs/_initiatives/tech-community-support.md [hunk 1]
```diff
@@ -2,4 +2,4 @@
 name: Tech Community Support
 ---
 
-We support a select number of local tech meetups by **providing a meeting venue** if needed and other organizational support. We also **give local tech meetups  a Discord channel on our TEC server**. This provides a virtual place to share documents, plan the next meetup and keep the conversation going. **[Contact us](/contact.html) to learn more.** 
+We support a select number of local tech meetups by **providing a meeting venue** if needed and other organizational support. We also **give local tech meetups  a Discord channel on our [TEC server](https://discord.gg/EDfSeTT2Jb)**. This provides a virtual place to share documents, plan the next meetup and keep the conversation going. **[Contact us](/contact.html) to learn more.**
```
Comment (cderr) [inline docs/_initiatives/tech-community-support.md]:
```suggestion
We support a select number of local tech meetups by **providing a meeting venue** if needed and other organizational support. We also **give local tech meetups  a Discord channel on our [TEC server](https://discord.gg/EDfSeTT2Jb)**. This provides a virtual place to share documents, plan the next meetup and keep the conversation going. **[Contact us](/contact.html) to learn more.**
```

File: docs/_people/wyatt_johnson.md [hunk 2]
```diff
@@ -10,7 +10,6 @@ sequence: 4
 Wyatt Johnson is the CNT/CIS Program Director at Gateway Community College in 
 Phoenix, AZ and teaches mainly Networking and Cybersecurity classes. He has a 
 Masters in Curriculum and Instruction from ASU, a decent sized stack of 
-certifications and years of IT experience. He firmly believes that Star Wars, 
-Star Trek and that Tom Baker was the best Doctor. One of his most disappointing
+certifications and years of IT experience. He firmly believes that Star Wars > Star Trek and that Tom Baker was the best Doctor. One of his most disappointing
 moments was finding out that there was no way to make watching Monty Python 
 and the Search for the Holy Grail a prerequisite for any class.
```
Comment (tiefps) [inline docs/_people/wyatt_johnson.md]: This line is pretty long. Please break it up to keep it roughly consistent with the others.

File: docs/_posts/2021-03-04-relinquishing-our-humanity.md [hunk 3]
```diff
@@ -12,7 +12,9 @@ image: assets/images/articles/relinquishing_our_humanity.jpg
 
 *The inadvertent consequence of well-intentioned technology*
 
-![man with VR headset holding a phone and entangled in wires]({{page.image}})
+![woman in a tank top using a VR headset surrounded by computer monitors]({{page.image}})
+Photo by [cottonbro studio](https://www.pexels.com/photo/a-woman-in-a-tank-top-using-a-vr-headset-8721318/)
```
Comment (tiefps) [inline docs/_posts/2021-03-04-relinquishing-our-humanity.md:16]: Thanks for remembering to updating the alt text! To be better about using it consistently, I created an issue to standardize it in front matter (https://github.com/techedcollab/techedcollab.github.io/issues/6).

File: docs/_people/wyatt_johnson.md [hunk 4]
```diff
@@ -10,7 +10,6 @@ sequence: 4
 Wyatt Johnson is the CNT/CIS Program Director at Gateway Community College in 
 Phoenix, AZ and teaches mainly Networking and Cybersecurity classes. He has a 
 Masters in Curriculum and Instruction from ASU, a decent sized stack of 
-certifications and years of IT experience. He firmly believes that Star Wars, 
-Star Trek and that Tom Baker was the best Doctor. One of his most disappointing
+certifications and years of IT experience. He firmly believes that Star Wars > Star Trek and that Tom Baker was the best Doctor. One of his most disappointing
```
Comment (tiefps) [inline docs/_people/wyatt_johnson.md]:
```suggestion
certifications and years of IT experience. He firmly believes that Star Wars > Star Trek
 and that Tom Baker was the best Doctor. One of his most disappointing
```

**Review by tiefps — APPROVED** [review]

**Pull request merged**